### DISCUSSION TITLE
developement state

### DISCUSSION THREAD

**Discussion by Ing-Dom** [sstaub/Ethernet3#58] [General]
Hi Stefan @sstaub ,

I currently use Ethernet_Generic from khoi_prog which seems to utilize this lib.
Ethernet_Generic is archived now, so I think switching over to Ethernet3. Additionally I would have some enhancement I would make PRs for that (SPI1, Serial logging interface, maybe more).
As there is a PR open for over 6 months, general question: Do you acceppt PRs in general?

**Comment by sstaub** [comment]
In general I accept PRs but in the moment I have no time to test it. So I can't say when I can accept it.
